packages/addons/service/nextpvr/source/bin/nextpvr-downloader

```diff
@@ -9,7 +9,8 @@ oe_setup_addon service.nextpvr
 ICON="${ADDON_DIR}/resources/icon.png"
 CONTROL_FILE="/tmp/curl.nextpvr.done"
 DATA_FILE="/tmp/curl.nextpvr.data"
-NEXTPVR_FILE="NPVR-@NEXTPVR_VERSION@.zip"
+readarray -td~ -c 1 -t version < <(printf '%s' "@NEXTPVR_VERSION@")
```

Comment:
is this construct just to transform `6.1.1~1` to `6.1.1` ?

untested
```
NEXTPVR_VERSION="@NEXTPVR_VERSION@"
NEXTPVR_FILE="NPVR-${NEXTPVR_VERSION%~*}.zip"
```

Reply:
Yes, but your solution is better, I didn't know that form.  Updated.